containers/starbunk-dnd/infrastructure/persistence/repositoryFactory.ts

```
@@ -0,0 +1,124 @@
+	static createCampaignRepository(): CampaignRepository {
+		return new MockCampaignRepository();
+	}
+
+	static createCharacterRepository(): CharacterRepository {
+		return new MockCharacterRepository();
+	}
+
+	static createGameSessionRepository(): GameSessionRepository {
+		return new MockGameSessionRepository();
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)